renderer/src/hooks/StationWallet.tsx

```diff
@@ -138,10 +143,10 @@ interface SplitTransactions {
 }
 
 const splitWalletTransactions = (transactions: (FILTransaction|FILTransactionProcessing)[]): SplitTransactions => {
-  const processing: FILTransactionProcessing | null =
-  transactions.find(tx => tx.status === 'processing') as FILTransactionProcessing
+  const processing: FILTransactionProcessing | undefined =
+  transactions.find(isFILTransactionProcessing)
```

Comment:
I think you don't need the explicit type declaration now.

```suggestion
  const processing = transactions.find(isFILTransactionProcessing)
```

Of course, feel free to keep it if you find it helpful to have the type of variable explicitly declared in the spruce code.